### ISSUE TITLE
False positive in DeprecatedTemplateBracketsRule

### ISSUE THREAD

**Issue by ericwb** [willthames/ansible-lint#59]
The DeprecatedTemplateBracketsRule module will report ANSIBLE0001 in the following example even though this would be a valid use of ${}

  lineinfile: dest=/etc/init.d/mysql regexp="^MYSQLD_STARTUP_TIMEOUT" insertafter="^### END INIT INFO" line="MYSQLD_STARTUP_TIMEOUT=${MYSQLD_STARTUP_TIMEOUT:-180}"

**Comment by willthames** [comment]
See #40 for to skip ansible-lint for this task

(I should update the docs though)

**Comment by ericwb** [comment]
This worked nicely for other rules, but for some reason it doesn't help with DeprecatedTemplateBracketsRule.

I have:
```
- name: increase timeout in mysql init file
  lineinfile: dest=/etc/init.d/mysql regexp="^MYSQLD_STARTUP_TIMEOUT" insertafter="^### END INIT INFO" line="MYSQLD_STARTUP_TIMEOUT=${MYSQLD_STARTUP_TIMEOUT:-180}"
  tags:
    - skip_ansible_lint
```
I still get:
```
[ANSIBLE0001] Old style (${var}) brackets
/ansible/roles/prep-db/tasks/main.yml:33
  lineinfile: dest=/etc/init.d/mysql regexp="^MYSQLD_STARTUP_TIMEOUT" insertafter="^### END INIT INFO" line="MYSQLD_STARTUP_TIMEOUT=${MYSQLD_STARTUP_TIMEOUT:-180}"

[ANSIBLE0001] Old style (${var}) brackets
/ansible/roles/prep-db/tasks/main.yml:33
  lineinfile: dest=/etc/init.d/mysql regexp="^MYSQLD_STARTUP_TIMEOUT" insertafter="^### END INIT INFO" line="MYSQLD_STARTUP_TIMEOUT=${MYSQLD_STARTUP_TIMEOUT:-180}"
```

**Comment by ericwb** [comment]
Ah, looks like skip_ansible_lint only works with matchtasks() function not match() which is used by DeprecatedTemplateBracketsRule.

**Comment by willthames** [comment]
ANSIBLE0001 has been removed from ansible-lint so this should no longer be an issue.
